openapi.yaml

```diff
@@ -613,6 +616,7 @@ definitions:
             required:
               - name
               - website
+            additionalProperties: false
```

Comment:
Also, `data` is required in `body`; `type` and `attributes` are required in the `data` object. Same with other post bodies. Similarly, patch bodies also need to specify these required fields, the only difference is `id` will be another required field in the patch body as well.

Example: https://github.com/osu-mist/notes-api/blob/master/openapi.yaml#L273-L280